### ISSUE TITLE
Issue with Linux Mint Software Manager

### ISSUE THREAD

**Issue by Kwakers01** [pwsafe/pwsafe#1416]
Your app is showing up on Linux Mint Cinnamon  (22 Wilma)  - Software Manager, but...

The size is looking rather large for some reason.

Name :  org.pwsafe.pwsafe
Version : v1.20.0
Size : 966 MB to download, 3.3 GB of disk space required
Branch : stable
Remote : Flathub

I though I would let you know in case you want to take a look and fix it.

I will down load from github as I do not want 3.3G of whatever (dependencies?) to be installed via the manager.

**Comment by ronys** [comment]
Looks like the flatpak version, which is typically larger.

Look for the .deb file.

**Comment by Kwakers01** [comment]
Hi,  Yes....  I am new-ish (back after 20 years) to Linux and just realised myself.  Enormous difference in size, I guess it downloads the flatpak software.  If I installed flatpak prior to installing any flatpaks, then would it download a smaller amount, or does it download gigs every time you download a flatpak.  It's something I need to look at.  Thanks anyway.

The software manager in linux mint shows version 1.20 as a flatpak (with the nice red password safe icon) and the non-flatpak (icon is a yellow folder and by someone else's name, not yours) is shown as v1.17 (I think).  I downloaded the v1.20 .deb from github and did a double click to install instead.

Thanks again for the reply.
